### ISSUE TITLE
File corrupted?

### ISSUE THREAD

**Issue by bpardue** [Hak4Kidz/H4K-badge-2019#8]
Hi, I was trying to help my son with the 2019 H4K Tyro badge. Whenever we tried to verify the code from GitHub (downloaded today) in Arduino, we constantly had stray character errors, and it looked very different than what was downloaded at the H4K earlier this year. It looks like someone may have tried to edit in a word processor, etc. (there are line break characters, etc.). Can a clean copy be made available again?

Thanks,
Bill Pardue

**Comment by Hak4Kidz** [comment]
The code looks good in github. I can upload a clean copy but I am skeptical that the code is the root cause. 

Before I upload a clean copy, can you please copy the raw code from github and paste it into a new file on your computer?

Did you know that the code for the DEF CON version of the badge has been published? https://github.com/Hak4Kidz/H4K-cryptex

Thanks,
Heal

**Comment by bpardue** [comment]
Thanks. It took a couple of tries. When copied/pasted directly into the
Arduino editor, I still received various character-related errors. When I
pasted into NotePad, that seemed to clean things up and I was finally able
to verify and upload. I needed to do that for both the Tyro Badge and Red
October files.

I've made sure to save a "clean" copies of tboth files, which we can go
back to, if needed.

--Bill
